cmd/ratify/cmd/utils.go

```diff
@@ -20,6 +20,8 @@ import (
 	"os"
 )
 
+const taggedReferenceWarning string = "Warning: Tagged references should NOT be used. The resolved digest may not point to the same signed artifact, since tags are mutable."
```

Comment:
```suggestion
const taggedReferenceWarning = "Warning: Tagged references should NOT be used. The resolved digest may not point to the same signed artifact, since tags are mutable."
```